examples/structured_data/time_series_anomaly_using_autoencoder.py

```diff
@@ -0,0 +1,344 @@
+"""
+Title: Timeseries anomaly detection using an Autoencoder
+Author: [pavithrasv](https://github.com/pavithrasv)
+Date created: 2020/05/31
+Last modified: 2020/05/31
+Description: Detect anomalies in a timeseries using an Autoencoder.
+"""
+
+"""
+## Introduction
+
+This script demonstrates how you can use a reconstruction convolutional autoencoder model
```

Comment:
Is there a reference you can provide for this model?

Reply:
I wanted to tryout timeseries anomaly detection, when reading about it i saw that autoencoders were used in the supervised solutions for this problem. I then played with LSTM/Conv1D architecture and arrived at this.